adapter/stdio/stdio.go

```diff
@@ -57,7 +58,18 @@ func (h *handler) HandleLogEntry(_ context.Context, instances []*logentry.Instan
 
 		for _, varName := range h.logEntryVars[instance.Name] {
 			if value, ok := instance.Variables[varName]; ok {
-				fields = append(fields, zap.Any(varName, value))
+				// TODO: remove when IP_ADDRESS is properly handled by Mixer
+				switch value.(type) {
```

Comment:
consider converting this to
`func field(varName string, value interface{}) zapcore.Field { }`
and reusing it in `HandleLogEntry` and `HandleMetrics`

Reply:
done.